src/components/shared/modal/Modal.tsx

```diff
@@ -9,7 +9,7 @@ import "./Modal.scss";
 export interface IModalProps {
   open: boolean;
   onOpen?: () => void;
```

Comment:
I think we can actually remove this from the code. I'm pretty sure its not used anywhere in the code.

Reply:
Well the tests are passing and I will say if you think so and the tests done break then throw it in the dumpster